src/devices/Ads1115/Ads1115.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
any reason for changing encoding?

Reply:
All files should be UTF8-BOM for consistency. I didn't do this one on purpose, but VS is apparently updating this automatically since the setting in .editorConfig is set like that.